src/css/components/panel.scss

```diff
@@ -24,12 +24,20 @@ h4 + .panel {
 
 .panel-title {
   font-size: 1.3rem;
-  font-weight: 600;
   margin-bottom: 2rem;
+
+  h1,
```

Comment:
Currently, panel title is applied directly to the `h` tag, so this shouldn't be needed.

```
<h1 className={ this.styler('panel-title') }>{ this.props.title }</h1>
```